src/dotnet-interactive/PackageDirectoryExtensionLoader.cs

```diff
@@ -12,11 +12,11 @@
 using Microsoft.DotNet.Interactive.Events;
 using Microsoft.DotNet.Interactive.Utility;
 using Pocket;
-using static Pocket.Logger<Microsoft.DotNet.Interactive.PackageDirectoryExtensionLoader>;
+using static Pocket.Logger<Microsoft.DotNet.Interactive.App.PackageDirectoryExtensionLoader>;
 
-namespace Microsoft.DotNet.Interactive;
+namespace Microsoft.DotNet.Interactive.App;
 
-internal class PackageDirectoryExtensionLoader : IKernelExtensionLoader
```

Comment:
how is this portable to .net repl and other custom app hosting .NET Interactive?

Reply:
Custom apps should implement their own extensibility mechanisms.

I think factoring this out into a separate small package is worthwhile so .NET REPL can continue to use it.

Reply:
Yes, totally in favour of a standalone package that could be opted in or implemented custom.